.appveyor.yml

```
@@ -0,0 +1,7 @@
+image: Visual Studio 2017
+matrix:
+    fast_finish: true
+
+build_script:
+    - cmake -H. -Bbuild
+    - cmake --build build
```

Add `--target` here so we only build what we need to build to game.
```suggestion
    - cmake --build build --target Sandbox
```